core/drivers/stm32_gpio.c

```diff
@@ -725,8 +734,30 @@ static TEE_Result dt_stm32_gpio_pinctrl(const void *fdt, int node,
 
 			STAILQ_INSERT_TAIL(&bank_list, bank, link);
 
-			if (IS_ENABLED(CFG_STM32MP13))
-				set_bank_gpio_non_secure(bank);
+			DMSG("Registered GPIO bank %c (%d pins) @%lx",
```

Comment:
```patch
-			DMSG("Registered GPIO bank %c (%d pins) @%lx",
+			DMSG("Registered GPIO bank %c (%u pins) @%#"PRIxVA,

Reply:
I'll reorder `dt-bindings: add TZPROT` macro and `drivers: stm32_gpio: add secure configuration for GPIOs` to ensure every patch compiles as well